.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.x
```

might wanna use the .nvmrc file from the root directory? Keeps your GHA build up to date with development environment
```suggestion
          node-version-file: .nvmrc
```